biomodels/src/test/java/org/reactome/release/ModelsTSVParserTester.java

```diff
@@ -2,68 +2,95 @@
 
 import org.junit.Test;
 
+import java.nio.file.Paths;
 import java.util.*;
 
-import static org.junit.Assert.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
 
 public class ModelsTSVParserTester {
 
     private final int correctPathwayCount = 3;
     private final int correctNumberPathwaysInTestFile = 745;
-    private final int correctNumberBioModelsIdsInTestFile = 153;
+    private final long correctNumberBioModelsIdsInTestFile = 153L;
+    private final String RESOURCE_DIR = Paths.get("src", "test", "resources").toFile().getAbsolutePath();
+
     @Test
-    public void emptyMapFromEmptyFileName() {
+    public void emptyFilenameReturnsEmptyMap() {
         Map<String, List<String>> pathwayToBioModelsIds =  ModelsTSVParser.parse("");
-        assertTrue("Empty file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty());
+        assertThat("Empty file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty(), is(true));
```

Comment:
A more standard way to write this may be:

`assertThat("Empty file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds, is(anEmptyMap()));`